dnf/query.py

```diff
@@ -44,6 +44,7 @@ def available(self):
         return self.filter(reponame__neq=hawkey.SYSTEM_REPO_NAME)
 
     def unneeded(self, sack, yumdb, debug_solver=False):
+        # :api
         goal = dnf.goal.Goal(sack)
```

Comment:
We discussed this with @jsilhan and decided not to include unneeded into api. It doesn't make sense without including also yumdb. Moreover, it was used only by the repoquery that has been already incorporated into DNF.